pkg/ruler/base/manager_metrics.go

```diff
@@ -1,4 +1,4 @@
-package ruler
+package base
 
 import (
 	"github.com/prometheus/client_golang/prometheus"
```

Comment:
The util package below could be avoided ?

Reply:
Nope, unfortunately. However, we are already working on forking `util`.